src/App.js

```diff
@@ -31,6 +31,8 @@ function App() {
 
   useEffect(() => {
     fetchWeather();
+    const interval = setInterval(fetchWeather, 5000);
```

Comment:
```suggestion
    const interval = setInterval(fetchWeather, 7000);
```
5 Seconds is really fast. I changed that to 7 seconds.